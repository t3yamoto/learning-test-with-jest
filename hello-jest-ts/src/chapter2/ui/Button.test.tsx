import renderer from "react-test-renderer";
import { Button } from "./Button";

describe('Button', () => {
    it('renderes correctly with react-test-renderer', () => {
        const button = renderer.create(<Button />);
        expect(button.toJSON()).toMatchSnapshot();
    })
})
