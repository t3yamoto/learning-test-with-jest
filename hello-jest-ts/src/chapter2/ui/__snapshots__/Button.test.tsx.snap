// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renderes correctly with react-test-renderer 1`] = `
<button
  onClick={[Function]}
>
  ON
</button>
`;
